tasks/hyrax-metadata-updates/index.js

```diff
@@ -101,21 +101,23 @@ async function getEntryTitle(config, metadata, isUmmG) {
 
   const searchParams = {
     short_name: shortName,
-    version: version,
+    version: versionId,
   };
 
   const result = await cmrInstance.searchCollections(searchParams);
-  // Verify that we have a valid result. If we don't then something is badly wrong
-  // and we should halt.
-  // Either the code is faulty or the provider is trying to ingest granules
+  // Verify that we have a valid result. If we don't then something is badly wrong and we
+  // should halt. Either the code is faulty or the provider is trying to ingest granules
   // into a collection that doesn't exist
-  const datasetId = get(result, '[0].dataset_id');
-
-  if (datasetId === undefined) {
-    throw new RecordDoesNotExist(`Unable to query parent collection entry title using short name ${shortName} and version ${version}`);
+  const conceptId = get(result, '[0].id');
+  shortName = get(result, '[0].short_name');
+  versionId = get(result, '[0].version_id');
+  if (conceptId === undefined) {
+    throw new RecordDoesNotExist(`Unable to query parent collection using short name ${shortName} and version ${versionId}`);
   }
 
-  return encodeURIComponent(datasetId);
+  return (config.addShortnameAndVersionIdToConceptId !== undefined
+    && config.addShortnameAndVersionIdToConceptId === true)
```

Comment:
Given that this conditional checks that `config.addShortnameAndVersionIdToConceptId` is both defined AND true, can't we just strict-check for true?